### ISSUE TITLE
Add a second view with signature information

### ISSUE THREAD

**Issue by mjbvz** [mattbierner/vscode-docs-view#2]
## Request
Add another view called `Signature Info` that shows the signature help (parameter hints) information at the current cursor position.

From: https://github.com/microsoft/vscode/issues/51253

**Comment by sandipchitale** [comment]
@mjbvz What is precisely meant by parameter info? Does **Documentation** view already shows that info? If I understand clearly what is different about **Signature information** which is not already covered by **Documentation**, I could submit a (almost ready) PR.

**Comment by mjbvz** [comment]
@sandipchitale Yep, the signature help is what I meant. Looks like you already found the right api to use for it too! Let me know if you run into any issues working on this

**Comment by sandipchitale** [comment]
Feel free to adjust the formatting. Note that it shows a green square next to active (current) signature.
